From 32736641e84ad374e6488f2456d5dd9279006812 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Thu, 24 Oct 2013 10:23:44 +0100
Subject: [PATCH] fdo#70553 lp#1231799: Fix empty quicklist, update .desktop
 format

replace libreoffice with UNIXBASISROOTNAME

df0fee94ef2c360ce509b34a2828022a8daf04d0

Change-Id: Ie036ac3055dfa15a4f80b898870385ec308c8c76
---
 sysui/desktop/menus/base.desktop    | 10 +++++-----
 sysui/desktop/menus/calc.desktop    | 10 +++++-----
 sysui/desktop/menus/draw.desktop    | 10 +++++-----
 sysui/desktop/menus/impress.desktop | 10 +++++-----
 sysui/desktop/menus/math.desktop    | 10 +++++-----
 sysui/desktop/menus/writer.desktop  | 10 +++++-----
 6 files changed, 30 insertions(+), 30 deletions(-)

Index: libreoffice-4.1.4~rc2/sysui/desktop/menus/base.desktop
===================================================================
--- libreoffice-4.1.4~rc2.orig/sysui/desktop/menus/base.desktop	2013-12-16 14:35:44.846933209 +0100
+++ libreoffice-4.1.4~rc2/sysui/desktop/menus/base.desktop	2013-12-16 14:35:44.838933210 +0100
@@ -30,10 +30,10 @@
 X-GIO-NoFuse=true
 Keywords=Data;SQL;
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=X-New
+Actions=NewDatabase
 StartupWMClass=libreoffice-base
 X-KDE-Protocols=file,http,smb,ftp,webdav
-[X-New Shortcut Group]
-UnityQuicklist=New Database
-Exec=libreoffice --base %%FILE%%
-TargetEnvironment=Unity
+[Desktop Action NewDatabase]
+Name=New Database
+Exec=${UNIXBASISROOTNAME} --base
+OnlyShowIn=Unity;
Index: libreoffice-4.1.4~rc2/sysui/desktop/menus/calc.desktop
===================================================================
--- libreoffice-4.1.4~rc2.orig/sysui/desktop/menus/calc.desktop	2013-12-16 14:35:44.846933209 +0100
+++ libreoffice-4.1.4~rc2/sysui/desktop/menus/calc.desktop	2013-12-16 14:35:44.838933210 +0100
@@ -30,10 +30,10 @@
 X-GIO-NoFuse=true
 Keywords=Accounting;Stats;OpenDocument Spreadsheet;Chart;Microsoft Excel;Microsoft Works;OpenOffice Calc;ods;xls;xlsx;
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=X-New
+Actions=NewSpreadsheet
 StartupWMClass=libreoffice-calc
 X-KDE-Protocols=file,http,smb,ftp,webdav
-[X-New Shortcut Group]
-UnityQuicklist=New Spreadsheet
-Exec=libreoffice --calc %%FILE%%
-TargetEnvironment=Unity
+[Desktop Action NewSpreadsheet]
+Name=New Spreadsheet
+Exec=${UNIXBASISROOTNAME} --calc
+OnlyShowIn=Unity;
Index: libreoffice-4.1.4~rc2/sysui/desktop/menus/draw.desktop
===================================================================
--- libreoffice-4.1.4~rc2.orig/sysui/desktop/menus/draw.desktop	2013-12-16 14:35:44.846933209 +0100
+++ libreoffice-4.1.4~rc2/sysui/desktop/menus/draw.desktop	2013-12-16 14:35:44.838933210 +0100
@@ -30,10 +30,10 @@
 X-GIO-NoFuse=true
 Keywords=Vector;Schema;Diagram;Layout;OpenDocument Graphics;Microsoft Publisher;Corel Draw;cdr;odg;svg;
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=X-New
+Actions=NewDrawing
 StartupWMClass=libreoffice-draw
 X-KDE-Protocols=file,http,smb,ftp,webdav
-[X-New Shortcut Group]
-UnityQuicklist=New Drawing
-Exec=libreoffice --draw %%FILE%%
-TargetEnvironment=Unity
+[Desktop Action NewDrawing]
+Name=New Drawing
+Exec=${UNIXBASISROOTNAME} --draw
+OnlyShowIn=Unity;
Index: libreoffice-4.1.4~rc2/sysui/desktop/menus/impress.desktop
===================================================================
--- libreoffice-4.1.4~rc2.orig/sysui/desktop/menus/impress.desktop	2013-12-16 14:35:44.846933209 +0100
+++ libreoffice-4.1.4~rc2/sysui/desktop/menus/impress.desktop	2013-12-16 14:35:44.838933210 +0100
@@ -30,10 +30,10 @@
 X-GIO-NoFuse=true
 Keywords=Slideshow;Slides;OpenDocument Presentation;Microsoft PowerPoint;Microsoft Works;OpenOffice Impress;odp;ppt;pptx;
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=X-New
+Actions=NewPresentation
 StartupWMClass=libreoffice-impress
 X-KDE-Protocols=file,http,smb,ftp,webdav
-[X-New Shortcut Group]
-UnityQuicklist=New Presentation
-Exec=libreoffice --impress %%FILE%%
-TargetEnvironment=Unity
+[Desktop Action NewPresentation]
+Name=New Presentation
+Exec=${UNIXBASISROOTNAME} --impress
+OnlyShowIn=Unity;
Index: libreoffice-4.1.4~rc2/sysui/desktop/menus/math.desktop
===================================================================
--- libreoffice-4.1.4~rc2.orig/sysui/desktop/menus/math.desktop	2013-12-16 14:35:44.846933209 +0100
+++ libreoffice-4.1.4~rc2/sysui/desktop/menus/math.desktop	2013-12-16 14:35:44.838933210 +0100
@@ -31,10 +31,10 @@
 X-GIO-NoFuse=true
 Keywords=Equation;OpenDocument Formula;Formula;odf;
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=X-New
+Actions=NewFormula
 StartupWMClass=libreoffice-math
 X-KDE-Protocols=file,http,smb,ftp,webdav
-[X-New Shortcut Group]
-UnityQuicklist=New Formula
-Exec=libreoffice --math %%FILE%%
-TargetEnvironment=Unity
+[Desktop Action NewFormula]
+Name=New Formula
+Exec=${UNIXBASISROOTNAME} --math
+OnlyShowIn=Unity;
Index: libreoffice-4.1.4~rc2/sysui/desktop/menus/writer.desktop
===================================================================
--- libreoffice-4.1.4~rc2.orig/sysui/desktop/menus/writer.desktop	2013-12-16 14:35:44.846933209 +0100
+++ libreoffice-4.1.4~rc2/sysui/desktop/menus/writer.desktop	2013-12-16 14:35:44.838933210 +0100
@@ -30,10 +30,10 @@
 X-GIO-NoFuse=true
 Keywords=Text;Letter;Fax;Document;OpenDocument Text;Microsoft Word;Microsoft Works;Lotus WordPro;OpenOffice Writer;CV;odt;doc;docx;rtf;
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=X-New
+Actions=NewDocument
 StartupWMClass=libreoffice-writer
 X-KDE-Protocols=file,http,smb,ftp,webdav
-[X-New Shortcut Group]
-UnityQuicklist=New Document
-Exec=libreoffice --writer %%FILE%%
-TargetEnvironment=Unity
+[Desktop Action NewDocument]
+Name=New Document
+Exec=${UNIXBASISROOTNAME} --writer
+OnlyShowIn=Unity;
